OpenRA.Mods.RA/Effects/GpsSatellite.cs

```diff
@@ -36,8 +41,12 @@ public void Tick(World world)
 			anim.Tick();
 			pos += new WVec(0, 0, 427);
 
-			if (pos.Z > pos.Y)
+			if (++tick >= revealDelay)
+			{
+				var owner = launcher.Owner.PlayerActor.Trait<GpsWatcher>();
```

Comment:
It would be even better if we could replace the `Actor launcher` ctor arg with a `Player owner`.  It looks like `GpsWatcher` doesn't care about the launcher for anything important, so this should hopefully be a simple change.